public/app/features/search/service/searcher.ts

```diff
@@ -13,7 +13,7 @@ export function getGrafanaSearcher(): GrafanaSearcher {
     const useBluge = config.featureToggles.panelTitleSearch;
     searcher = useBluge ? new BlugeSearcher(sqlSearcher) : sqlSearcher;
 
-    if (useBluge && location.search.indexOf('do-frontend-query')) {
+    if (useBluge && location.search.indexOf('do-frontend-query') !== -1) {
```

Comment:
Does `location.search.includes()` also work? That more clearly states your intention.

Reply:
yep thats better!